backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/impl/InOutProducer.java

```diff
@@ -527,6 +527,8 @@ private final I_M_InOut createReceiptHeader(final I_M_ReceiptSchedule rs)
 			receiptHeader.setExternalResourceURL(getExternalResourceURL(rs));
 		}
 
+		receiptHeader.setPOReference(rs.getPOReference());
```

Comment:
i think we shall also and the POReference column to header aggregation key
see https://github.com/metasfresh/metasfresh/blob/0333694fa227deeb6ac0f730c459df5380439602/backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/modelvalidator/ReceiptScheduleValidator.java#L88

Reply:
done